packages/logs/test/logs.entry.spec.ts

```diff
@@ -11,6 +12,7 @@ describe('logs entry', () => {
 
   afterEach(() => {
     stopSessionManagement()
+    startXhrProxy().reset()
```

Comment:
This is a bit confusing. Maybe we could have a standalone function `stopXhrProxy()` or `resetXhrProxy()`? Also this is not obvious where it is started in this file, maybe add a comment? (same for rum.entry.spec.ts)

Reply:
yes, it could be nicer